src/drivers/mct/cime_config/config_component.xml

```diff
@@ -509,7 +517,7 @@
   <entry id="BATCH_SYSTEM">
      <type>char</type>
      <default_value>none</default_value>
-     <valid_values>lc_slurm,moab,pbs,lsf,slurm,cobalt,cobalt_theta,none</valid_values>
+     <valid_values>nersc_slurm,lc_slurm,moab,pbs,lsf,slurm,cobalt,cobalt_theta,none</valid_values>
```

Comment:
Why so many exceptions?  All on the acme side...

Reply:
@jedwards4b , that's a good question. I was not involved in the creation of any of these. @ndkeen , do you know why a nersc_slurm batch system was created? It appears to be an exact copy of slurm.

Reply:
Correction, it looks like nersc_slurm uses -q to select queue and slurm uses -p.

Reply:
We do that on the cesm side and still maintain a single slurm definition.

Reply:
You're right, we need to clean our stuff up.